topo_processor/util/epsg.py

```diff
@@ -0,0 +1,6 @@
+from rasterio.crs import CRS
+
+
+def epsg_code(crs: CRS):
+    if crs.is_epsg_code:
+        return int(crs["init"].lstrip("epsg:"))
```

Comment:
there has to be a better way of getting a epsg code from a epsg object what do the docs recommend?

Reply:
crs = CRS.from_string('EPSG:3005')